charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
           image: "{{ .Values.manager.image.repository }}:{{ .Values.manager.image.tag }}"
           name: manager
           ports:
+            - containerPort: {{ .Values.manager.ports.metricsPort }}
```

Comment:
I don't see `.Values.manager.ports.metricsPort` in the default values.  What happens if the user doesn't manually add it in their values.yaml?  How do they know to add it?

Reply:
NVM, I wasn't looking at main on accident.